blockreassurance.php

```diff
@@ -19,339 +19,378 @@
-                if (isset($_FILES['info'.$i.'_file']) && isset($_FILES['info'.$i.'_file']['tmp_name']) && !empty($_FILES['info'.$i.'_file']['tmp_name'])) {
-                    if ($error = ImageManager::validateUpload($_FILES['info'.$i.'_file'])) {
-                        return false;
-                    } elseif (!($tmpName = tempnam(_PS_TMP_IMG_DIR_, 'PS')) || !move_uploaded_file($_FILES['info'.$i.'_file']['tmp_name'], $tmpName)) {
-                        return false;
-                    } elseif (!ImageManager::resize($tmpName, dirname(__FILE__).'/img/'.$filename[0].'.jpg')) {
-                        return false;
-                    }
-                    unlink($tmpName);
-                }
-                Db::getInstance()->execute('INSERT INTO `'._DB_PREFIX_.'reassurance` (`filename`,`text`)
-                                            VALUES ("'.((isset($filename[0]) && $filename[0] != '') ? pSQL($filename[0]) : '').
-                    '", "'.((isset($_POST['info'.$i.'_text']) && $_POST['info'.$i.'_text'] != '') ? pSQL($_POST['info'.$i.'_text']) : '').'")');
-            }
-            return true;
-        } else {
-            return false;
-        }
+        return is_writable($this->folder_file_upload)
+            && is_executable($this->folder_file_upload);
     }
 
-    public function removeFromDB()
+    /**
+     * getContent
+     *
+     *
+     * @return template
+    */
+    public function getContent()
     {
-        $dir = opendir(dirname(__FILE__).'/img');
-        while (false !== ($file = readdir($dir))) {
-            $path = dirname(__FILE__).'/img/'.$file;
-            if ($file != '..' && $file != '.' && !is_dir($file)) {
-                unlink($path);
-            }
+        $this->loadAsset();
+
+        $id_lang = $this->context->language->id;
+        $currentPage = Tools::getValue('page');
+
+        // get current page
+        if ($currentPage != 'global') {
+             $currentPage = 'global';
         }
-        closedir($dir);
 
-        return Db::getInstance()->execute('DELETE FROM `'._DB_PREFIX_.'reassurance`');
+        $moduleAdminLink = Context::getContext()->link->getAdminLink('AdminModules', true).'&configure='.$this->name.'&module_name='.$this->name;
+
+        $allCms = CMS::listCms($id_lang);
+
+        $this->context->smarty->assign(array(
+            'psr_hook_header' => Configuration::get('PSR_HOOK_HEADER'),
+            'psr_hook_footer' => Configuration::get('PSR_HOOK_FOOTER'),
+            'psr_hook_product' => Configuration::get('PSR_HOOK_PRODUCT'),
+            'psr_hook_checkout' => Configuration::get('PSR_HOOK_CHECKOUT'),
+            'psr_text_color' => Configuration::get('PSR_TEXT_COLOR'),
+            'psr_icon_color' => Configuration::get('PSR_ICON_COLOR'),
+            'logo_path' => $this->logo_path,
+            'guide_link' => $this->ps_url.'modules/'.$this->name.'/docs/doc_psreassurance_'.$this->context->language->iso_code.'.pdf',
```

Comment:
If this module is going to be shipped with PS 1.7.6, then we are going to have people from all over the world using it 🌏 . So does this link provide docs for China for example ? 😄 

If not, we should check whether the language is available for the FAQ. By the way, who will write it and maintain it then ?

Reply:
there is no help tab, remove this line